S/Spasm/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spasm"
+version = v"1.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/laurentbartholdi/spasm.git", "072719a40c837e447dfe4ae9e4941c60d9a28eda"),
+    GitSource("https://github.com/linbox-team/givaro.git", "fc6cac7820539c900dde332326c71461ba7b910b"),
+    GitSource("https://github.com/linbox-team/fflas-ffpack.git", "94aa88263f5c6032adb5c86bf806f007fec7aded"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir
+
+env
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p0 ${f}
+done
+
+cd ${WORKSPACE}/srcdir/givaro
+./autogen.sh CCNAM=gcc --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+
+cd ${WORKSPACE}/srcdir/fflas-ffpack
+./autogen.sh PKG_CONFIG_PATH=$prefix/lib/pkgconfig --prefix=$prefix --build=${MACHTYPE} --host=${target}`)
```

Comment:
`` `) `` loks like an error?

Reply:
Indeed! However, the compilation never progressed beyond Givaro, because it couldn't detect gmp properly